docker.md

```diff
@@ -26,32 +29,42 @@ sudo pacman -Sy docker
 
 ---
 
-## Build the YunoHost container
+## Get the YunoHost image
 
-The following command will fetch the latest YunoHost built image:
+The following command will fetch the latest YunoHost built image for **amd64**:
 ```bash
-docker pull zamentur/yunohost-stable8
+docker pull domainelibre/yunohost
 ```
 
-You can also build the image manually:
+The following command will fetch the latest YunoHost built image for **arm v7** (ex : Raspberry PI 2):
 ```bash
-docker build -t zamentur/yunohost-stable8 github.com/YunoHost/Dockerfile
+docker pull domainelibre/yunohost-arm
 ```
 
-You can check that the container is successfully built with the `docker images` command.
+## Build the YunoHost image
+
+You can also build the image manually:
+
+Please follow *Building* section [here](https://github.com/aymhce/YunohostDockerImage)
 
 ---
 
 ## Run the container
+
+> **The linux docker host must run systemd**
+
+> **WARN : --privileged mod is required for systemd**
```

Comment:
YunoHost should detect it is install in docker to disable firewall operation. Without that if you run --priviledge, each app or domain install will redefine firewall rules of the host.

For me it is a very important point  to fix. Without I am against to speak about an official way to run yunohost with docker.